docs/testing/debug-desktop-using-edge-chromium.md

```diff
@@ -33,18 +34,18 @@ These instructions assume you have experience using the command line, understand
 
    - If you want to create a project to experiment with debugging in Visual Studio Code, use the [Yeoman generator for Office Add-ins](../develop/yeoman-generator-overview.md). Use any one of our quick start guides, such as the [Outlook add-in quick start](../quickstarts/outlook-quickstart.md), in order to do this.
    - If you want to debug an existing project that was created with Yo Office, skip to the next step.
-   - If you want to debug an existing project that was not created with Yo Office, complete the procedure in the [Appendix A](#appendix-a) and then return to the next step of this procedure.
+   - If you want to debug an existing project that was not created with Yo Office, complete the procedure in the [Debug a project not created with yo office](#debug-a-project-not-created-with-yo-office) and then return to the next step of this procedure.
```

Comment:
The text before the numbered steps mentions a yo office project as a prerequisite. Seems confusing to have the instructions about what to do if that isn't the case embedded later.

Reply:
I removed this step and modified the titles to indicate there are 2 sections (yo office, and non-yo office)